components/brave_wallet/browser/zcash/zcash_wallet_service.cc

```diff
@@ -531,4 +657,22 @@ void ZCashWalletService::Unlocked() {
   }
 }
 
+void ZCashWalletService::Locked() {
+#if BUILDFLAG(ENABLE_ORCHARD)
+  for (const auto& pair : shield_sync_services_) {
+    pair.second->PauseSyncing();
```

Comment:
Let's not introduce now paused/stopped state but just remove running syncers on lock

Reply:
Done